Mac OS does not correctly support the math library long double constants

This causes and issue compiling the ld-decode-tools ld-comb-ntsc on mac:

comb.cpp:732:83: error: use of undeclared identifier 'M_PIl'; did you mean
       'P_PID'?
   ...double>(y), x) * (180 / M_PIl));
                              ^~~~~
                              P_PID

This can be fixed by defining the constant by hand:

#ifndef M_PIl
#define M_PIl       0xc.90fdaa22168c235p-2L
#endif

--- a/tools/ld-chroma-decoder/comb.h
+++ b/tools/ld-chroma-decoder/comb.h
@@ -40,12 +40,6 @@
 #include "yiq.h"
 #include "yiqbuffer.h"
 
-// Fix required for Mac OS compilation - environment doesn't seem to set up
-// the expected definitions properly
-#ifndef M_PIl
-#define M_PIl 0xc.90fdaa22168c235p-2L
-#endif
-
 class Comb
 {
 public:

